src/utils/dummyData.js

```diff
@@ -0,0 +1,254 @@
+const players = [
```

Comment:
So may changes before the commit. Good idea to have smaller changes you can keep track of and a number of commits to accompany the changes.